src/core/Form/Radiobutton/Radiobutton.baseStyles.tsx

```diff
@@ -0,0 +1,159 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { disabledCursor } from '../../../components/utils/css';
+import { element, font } from '../../theme/reset';
+import { focus } from '../../theme/utils/focus';
+
+/* stylelint-disable no-descending-specificity */
+export const baseStyles = withSuomifiTheme(
+  ({ theme }: TokensAndTheme) => css`
+    ${element({ theme })}
+    ${font({ theme })('bodyText')}
+
+  &.fi-radiobutton {
+    & .fi-radiobutton_hintText {
+        display: block;
+        padding-left: 26px;
+        color: ${theme.colors.depthDark1};
+        ${theme.typography.bodyTextSmall};
+    }
+
+    & .fi-radiobutton_input {
+      opacity: 0;
+      position: absolute;
+      margin: 8px 0 0 3px;
+
+      + label {
```

Comment:
Not using component specific names in every place on these scss rules (e.g **_fi-radiobutton_label_**), but should we?

Reply:
I'd assume it's more risky to do it this way, but since we are only supporting strings as children I guess there is no issue?

Reply:
I see no big threat either, but somehow would still feel safer with using specific classnames. Would be clearer for developers as well.

Reply:
These all are now changed to be explicitly named, so the example shown above is now

https://github.com/vrk-kpa/suomifi-ui-components/blob/f2c4662246ea8313f769cbafa2d9c9006732874f/src/core/Form/Radiobutton/Radiobutton.baseStyles.tsx#L23-L26